### ISSUE TITLE
add config file

### ISSUE THREAD

**Issue by ghost** [zk1931/pulsefs#56]
users need to be able to configure pulsed and its logging via config file.

just as a reference, here is the etcd config format: https://github.com/coreos/etcd/blob/master/Documentation/configuration.md

**Comment by EasonLiao** [comment]
Do you have any preference about the format of configuration file?

**Comment by ghost** [comment]
yes how about ini file?

http://commons.apache.org/proper/commons-configuration/javadocs/v1.10/apidocs/index.html?org/apache/commons/configuration/HierarchicalINIConfiguration.html

**Comment by EasonLiao** [comment]
Sounds great, I was planning work on it today but I just saw this is for 0.2.0, so I guess I'll work on something else today.

**Comment by ghost** [comment]
2 features i'd like to have before releasing 0.1.0 is batch and ephemeral files.

**Comment by EasonLiao** [comment]
Yes that's more important, I'll try to finish ephemeral node today.
